### PR TITLE
Telemetry log events

### PR DESCRIPTION
**Pull request by Sense**

### CODE REVIEW

File: src/panel/category_panel.js [hunk 1]
```diff
@@ -111,10 +113,9 @@ export default class CategoryPanel {
   }
 
   selectPoi(poi){
-    // TODO telemetry
     fire('fit_map', poi, layouts.LIST)
     this.close(false)
-    PanelManager.loadPoiById(poi.id, {isFromList : true, list: this})
+    PanelManager.loadPoiById(poi.id, {isFromList : true, list: this, source: 'pagesjaunes'})
```
Comment (mlnor27) [inline src/panel/category_panel.js]: Wouldn't it be better if we create a constant object for the poi sources ?

File: src/panel/poi_panel.js [hunk 2]
```diff
@@ -139,12 +149,20 @@ PoiPanel.prototype.setPoi = async function (poi, options = {}) {
 }
 
 PoiPanel.prototype.center = function() {
-  Telemetry.add(Telemetry.POI_GO)
+  if (this.poiSource && this.poiSource == 'pagesjaunes') {
```
Comment (mlnor27) [inline src/panel/poi_panel.js]: Following my last comment, now that i saw more code, a constant should be used to reference the poi sources 👍

File: src/panel/category_panel.js [hunk 3]
```diff
@@ -111,10 +114,9 @@ export default class CategoryPanel {
   }
 
   selectPoi(poi){
-    // TODO telemetry
     fire('fit_map', poi, layouts.LIST)
     this.close(false)
-    PanelManager.loadPoiById(poi.id, {isFromList : true, list: this})
+    PanelManager.loadPoiById(poi.id, {isFromList : true, list: this, source: constants.telemetry.source.PJ})
```
Comment (amatissart) [inline src/panel/category_panel.js]: `source` should be read from the API response. The category panel may be filled with POIs from different sources (depending on the location, the category, etc.)

Comment (Sense) [reply src/panel/category_panel.js]: Ok @amatissart, I don't see any `source` attribute coming from the API response in the IdunnPOI object. I don't think extracting it from the `id` would be a sustainable solution.

File: src/panel/category_panel.js (hunk 1 above)
Comment (Sense) [reply src/panel/category_panel.js]: Indeed, done

File: src/panel/poi_panel.js (hunk 2 above)
Comment (Sense) [reply src/panel/poi_panel.js]: We're currently talking about using the API response to set the poiSource.

**Pull request merged**